gradle.properties

```diff
@@ -12,3 +12,6 @@ jacksonVersion=1.8.3
 luceneVersion=3.6.0
 awsVersion=1.9.3
 mockitoVersion=1.8.5
+jodaTimeVersion=2.9.9
+
+m2SettingsFile=~/.m2/settings.xml
```

Comment:
This is not really safe. many build environments don't pull from `~/.m2` and `~/.m2` is not guaranteed to be the maven directory.